store.go

```diff
@@ -166,6 +167,26 @@ type flaggableStore interface {
 
 type StoreOptions = types.StoreOptions
 
+type HashMethod = dedup.HashMethod
+
+const (
+	HashInvalid  = dedup.HashInvalid
```

Comment:
`DedupHash…` for all of this, please, to reduce ambiguity within the large `storage` namespace.